### PR TITLE
[KO] Update `People/osu! Team`

### PR DESCRIPTION
**Pull request by Sitoria**
## Self-check

- [x] The changes are tested against the [contribution checklist](https://osu.ppy.sh/wiki/osu!_wiki/Contribution_guide#self-check)
- [x] @sylphi 
- [x] [whats the difference between `moderator`  and `GMT`](https://discord.com/channels/188630481301012481/218677502141399041/1143371777754537984) --> [Answer](https://discord.com/channels/188630481301012481/218677502141399041/1143450938410012682)
- [x] *(translations only)* The changes are reviewed on GitHub [by a fluent speaker](https://osu.ppy.sh/wiki/osu!_wiki/Contribution_guide#review)

### CODE REVIEW

File: wiki/People/osu!_team/ko.md [hunk 1]
```diff
@@ -21,59 +19,73 @@ outdated_since: 6b280c8bceae0f3d6fd6d62fb8e47f060a632106
...
+| ::{ flag=CA }:: [Azer](https://osu.ppy.sh/users/2155578) | [월드컵](/wiki/Tournaments#official-world-cups) 주최자 및 관리자 |
 | ::{ flag=US }:: [ChillierPear](https://osu.ppy.sh/users/9501251) | 월드컵 주최자 및 관리자 |
+| ::{ flag=US }:: [Death](https://osu.ppy.sh/users/3242450) | [모더레이터](/wiki/People/Global_Moderation_Team), 기술적 지원 |
+| ::{ flag=US }:: [Dntm8kmeeatu](https://osu.ppy.sh/users/5428812) | 모더레이터, 기술적 지원 |
+| ::{ flag=SK }:: [Domco](https://osu.ppy.sh/users/3562660) | osu! 개발자 |
 | ::{ flag=AU }:: [Ephemeral](https://osu.ppy.sh/users/102335) | 프로젝트 및 커뮤니티 유지 관리자, 공식 아티스트 지원 |
 | ::{ flag=JP }:: [flyte](https://osu.ppy.sh/users/3103765) | 디자인 책임자, Creative Cloud 상시 감시자 |
-| ::{ flag=TN }:: [Hivie](https://osu.ppy.sh/users/14102976) | 모딩 및 매핑 자문가 |
+| ::{ flag=SA }:: [frenzibyte](https://osu.ppy.sh/users/14210502) | osu! 개발자 |
+| ::{ flag=TN }:: [Hivie](https://osu.ppy.sh/users/14102976) | [노미네이션 어세스먼트 팀](/wiki/People/Nomination_Assessment_Team) 리더, [비트맵 스포트라이트](/wiki/Beatmap_Spotlights) 메니저 |
```
[the remark this answers is not in the document]
Comment (Sitoria) [reply wiki/People/osu!_team/ko.md:30]: applied

File: wiki/People/osu!_team/ko.md [hunk 2]
```diff
@@ -21,59 +19,73 @@ outdated_since: 6b280c8bceae0f3d6fd6d62fb8e47f060a632106
...
 | ::{ flag=US }:: [radar](https://osu.ppy.sh/users/7131099) | 모딩 및 매핑 자문가 |
 | ::{ flag=MX }:: [Repflez](https://osu.ppy.sh/users/201392) | 총괄 관리자, 위키 관리자 |
 | ::{ flag=FR }:: [Shiro](https://osu.ppy.sh/users/113005) | 총괄 관리자 |
 | ::{ flag=AU }:: [smoogipoo](https://osu.ppy.sh/users/1040328) | osu! 개발자, osu!mania 애호가, 버그 관련 담당자 |
+| ::{ flag=PL }:: [spaceman_atlas](https://osu.ppy.sh/users/3035836) | osu! 개발자 |
+| ::{ flag=FR }:: [ThePooN](https://osu.ppy.sh/users/718454) | osu! 인프라 개발자 |
 | ::{ flag=ES }:: [Trosk-](https://osu.ppy.sh/users/3469385) | 총괄 관리자, 토너먼트 보조 |
-| ::{ flag=PL }:: [Venix](https://osu.ppy.sh/users/5999631) | 중재 및 커뮤니티 자문가 |
+| ::{ flag=US }:: [tybug](https://osu.ppy.sh/users/12092800) | 총괄 관리자, 빗자루 청소부 |
+| ::{ flag=PL }:: [Venix](https://osu.ppy.sh/users/5999631) | [글로벌 모더레이션 팀](/wiki/People/Global_Moderation_Team) 리더, 비트맵 스포트라이트 메니저, Hivie의 비서 |
```
[the remark this answers is not in the document]
Comment (Sitoria) [reply wiki/People/osu!_team/ko.md:49]: im so sorry walavouchey ;w;

File: wiki/People/osu!_team/ko.md [hunk 3]
```diff
@@ -21,59 +19,73 @@ outdated_since: 6b280c8bceae0f3d6fd6d62fb8e47f060a632106
...
 | ::{ flag=AU }:: [Zallius](https://osu.ppy.sh/users/55) | 멸종 위기종 |
 
 상기 인원 이외에도, [계정 지원팀](/wiki/People/Account_support_team)이 손에 닿지 않는 부분에 대해 도움을 드리고 있습니다.
 
+핵심 osu! 팀 이외에도, 게임을 계속 운영하기 위해 [많은 자원 봉사 팀들](/wiki/People)이 있습니다.
```
[the remark this answers is not in the document]
Comment (Sitoria) [reply wiki/People/osu!_team/ko.md:54]: ok

File: wiki/People/osu!_team/ko.md [hunk 4]
```diff
@@ -21,59 +19,73 @@ outdated_since: 6b280c8bceae0f3d6fd6d62fb8e47f060a632106
...
+
 ## 활동 중이 아닌 팀원들
 
 | 이름 | 역할 |
 | :-- | :-- |
-| ::{ flag=CA }:: [awp](https://osu.ppy.sh/users/2650) | 총괄 관리자 및 도우미, Chibi 아티스트. [Barrel Roll Weekly](http://brw.twinkfish.com/) |
-| ::{ flag=US }:: [Chaos](https://osu.ppy.sh/users/2628870) | 중재 및 커뮤니티 자문가 |
-| ::{ flag=NO }:: [cYsmix](https://osu.ppy.sh/users/272870) | 오디오 디자이너, [공식 osu! 아티스트](https://osu.ppy.sh/beatmaps/artists/2) |
+| ::{ flag=CA }:: [awp](https://osu.ppy.sh/users/2650) | 총괄 관리자 및 도우미, Chibi 아티스트. |
```
[the remark this answers is not in the document]
Comment (Sitoria) [reply wiki/People/osu!_team/ko.md:60]: removed

File: wiki/People/osu!_team/ko.md [hunk 5]
```diff
@@ -21,59 +19,73 @@ outdated_since: 6b280c8bceae0f3d6fd6d62fb8e47f060a632106
...
+| ::{ flag=US }:: [Chaos](https://osu.ppy.sh/users/2628870) | 모더레이터, 커뮤니티 자문가 |
+| ::{ flag=NO }:: [cYsmix](https://osu.ppy.sh/users/272870) | 오디오 디자이너, [공식 아티스트](https://osu.ppy.sh/beatmaps/artists/2) |
+| ::{ flag=US }:: [Daru](https://osu.ppy.sh/users/32480) | osu!페인터 |
```
[the remark this answers is not in the document]
Comment (Sitoria) [reply wiki/People/osu!_team/ko.md:63]: reverted

File: wiki/People/osu!_team/ko.md [hunk 6]
```diff
@@ -21,59 +19,73 @@ outdated_since: 6b280c8bceae0f3d6fd6d62fb8e47f060a632106
...
 | ::{ flag=NZ }:: [deadbeat](https://osu.ppy.sh/users/128370) | 월드컵 주최자 및 관리자 |
 | ::{ flag=US }:: [Derekku](https://osu.ppy.sh/users/91341) | 총괄 관리자, 커뮤니티 매니저 |
-| ::{ flag=NZ }:: [Echo](https://osu.ppy.sh/users/431) | osu! 개발자, 인게임 채팅의 IRC 통합 기능 제공자, 사이트 관리자. [블로그](http://blog.echo.sh/) |
+| ::{ flag=FR }:: [Damnae](https://osu.ppy.sh/users/989377) | 스토리보드 개발자 |
+| ::{ flag=NZ }:: [Echo](https://osu.ppy.sh/users/431) | osu! 개발자, 인게임 채팅의 IRC 통합 기능 제공자, 사이트 관리자. |
```
[the remark this answers is not in the document]
Comment (Sitoria) [reply wiki/People/osu!_team/ko.md:67]: deleted

File: wiki/People/osu!_team/ko.md [hunk 7]
```diff
@@ -21,59 +19,73 @@ outdated_since: 6b280c8bceae0f3d6fd6d62fb8e47f060a632106
...
 | ::{ flag=US }:: [HappyStick](https://osu.ppy.sh/users/256802) | osu! Coffee Hour 호스트, 월드컵 스트리머, 토너먼트 위원 |
 | ::{ flag=NL }:: [Intermezzo](https://osu.ppy.sh/users/136842) | osu! 개발자, osz2 및 p2p 백엔드 제공자 |
-| ::{ flag=GB }:: [JBHyperion](https://osu.ppy.sh/users/4879508) | 모딩 및 매핑 자문가 |
-| ::{ flag=US }:: [Datawalke](https://osu.ppy.sh/users/142) | 원본 사이트 디자이너, 초기 호스팅 공급자. [Brave New Games](http://www.bravegamer.com/) |
+| ::{ flag=GB }:: [JBHyperion](https://osu.ppy.sh/users/4879508) |  전 [QAT](/wiki/People/Quality_Assurance_Team/QAT_Leaders) 리더 |
+| ::{ flag=US }:: [Datawalke](https://osu.ppy.sh/users/142) | 원본 사이트 디자이너, 초기 호스팅 공급자. |
```
[the remark this answers is not in the document]
Comment (Sitoria) [reply wiki/People/osu!_team/ko.md:71]: okay

**Review by Sitoria — COMMENTED** [review]
applied!

**Comment by Sitoria** [comment]
ready to merge ✅

**Review by TicClick — APPROVED** [review]

**Pull request merged**